ui/jest.config.js

```diff
@@ -0,0 +1,12 @@
+const nextJest = require('next/jest')
+
+const createJestConfig = nextJest({
+  dir: './',
+})
+
+const customJestConfig = {
+  moduleDirectories: ['node_modules', '<rootDir>/'],
+  testEnvironment: 'jest-environment-jsdom',
+}
+
+module.exports = createJestConfig(customJestConfig)
```

Comment:
```suggestion
module.exports = createJestConfig(customJestConfig)

```

Reply:
Done!, I was ignoring `.js` files, now I enabled `eslint` for those as well.